### PR TITLE
Feature/jr2 96 ui translation exporter importer

### PR DESCRIPTION
**Pull request by birom**

### CODE REVIEW

File: gradle/translation.gradle [hunk 1]
```diff
@@ -0,0 +1,137 @@
+apply plugin: "java"
```
Comment (michael-wirth) [inline gradle/translation.gradle]: do we really need the "java" plugin for these translation tasks? Remove if not required

File: src/main/webapp/i18n/de/configuration.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
         "table": {
-            "prefix": "Präfix",
+            "prefix": "Pr\u00e4fix",
             "properties": "Eigenschaften"
-        }
+        },
+        "title": "Konfiguration"
     }
-}
+}
```
Comment (michael-wirth) [inline src/main/webapp/i18n/de/configuration.json]: Is it possible to add am empty line at the end of the json files to satisfy github?

File: gradle/translation.gradle (hunk 1 above)
Comment (birom) [inline gradle/translation.gradle]: No we do not need it. I have removed it.

File: src/main/webapp/i18n/de/configuration.json (hunk 2 above)
Comment (birom) [inline src/main/webapp/i18n/de/configuration.json]: Done

**Comment by michael-wirth** [comment]
Approved.

Is the import/export documented?

**Pull request merged**